src/VisualStudio/Setup/VisualStudioSetup.csproj

```diff
@@ -221,6 +221,7 @@
     <NuGetPackageToIncludeInVsix Include="System.Composition.TypedParts" />
     <NuGetPackageToIncludeInVsix Include="System.Composition.Convention" />
     <NuGetPackageToIncludeInVsix Include="System.Composition.Hosting" />
+    <NuGetPackageToIncludeInVsix Include="ICSharpCode.Decompiler" />
```

Comment:
Does this also need NuGetPackageToIncludeInVsix Include="Mono.Cecil" as well?

Reply:
➡️ No. Both assemblies appear in the same package.

Reply:
Got it. That'll blow up in our face if ever we need to ship a second component who wants a different version, but fine for now. :-)